README.md

```diff
@@ -38,6 +35,10 @@ Prisma is a **next-generation ORM** that consists of these tools:
 
 Prisma Client can be used in _any_ Node.js or TypeScript backend application (including serverless applications and microservices). This can be a [REST API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/rest), a [GraphQL API](https://www.prisma.io/docs/concepts/overview/prisma-in-your-stack/graphql), a gRPC API, or anything else that needs a database.
 
+**The Prisma ORM can also further be extended with these Prisma solutions:**
```

Comment:
"Prisma solution"? Not "Prisma products"?
```suggestion
**The Prisma ORM can also further be extended with these Prisma products:**
```

Reply:
I had that first, more than happy with it :)